modules/grafana/templates/dashboards/deployment_panels/_links.json.erb

```diff
@@ -1,5 +1,5 @@
 {
-  "content": "<ul style=\"font-size: 20px;\">\n<li><a target=\"_blank\" href=\"https://sentry.io/govuk/app-<%= @app_name %>/\">Errors for <%= @app_name %> (Sentry)</a></li>\n<li><a target=\"_blank\" href=\"https://kibana.logit.io\">Kibana (Logit)</a></li>\n<li><a target=\"_blank\" href=\"https://alert.<%= @app_domain %>/\">Icinga</a></li>\n<li><a target=\"_blank\" href=\"https://docs.publishing.service.gov.uk/apps/<%= @docs_name %>.html\"><%= @app_name %> documentation</a></li>\n<li><a target=\"_blank\" href=\"https://govuk.zendesk.com/agent/filters/135081425\">Dashboard feedback</a></li>\n</ul>\n",
+  "content": "<ul style=\"font-size: 20px;\">\n<li><a target=\"_blank\" href=\"https://sentry.io/govuk/app-<%= @app_name %>/?query=is%3Aunresolved+environment%3A<%= hiera('govuk::deploy::config::errbit_environment_name') %>\">Errors for <%= @app_name %> (Sentry)</a></li>\n<li><a target=\"_blank\" href=\"https://kibana.logit.io\">Kibana (Logit)</a></li>\n<li><a target=\"_blank\" href=\"https://alert.<%= @app_domain %>/\">Icinga</a></li>\n<li><a target=\"_blank\" href=\"https://docs.publishing.service.gov.uk/apps/<%= @docs_name %>.html\"><%= @app_name %> documentation</a></li>\n<li><a target=\"_blank\" href=\"https://govuk.zendesk.com/agent/filters/135081425\">Dashboard feedback</a></li>\n</ul>\n",
```

Comment:
Is this even how you do this?

Reply:
I'm using that var because it's the one that is used to set [SENTRY_CURRENT_ENV env vars in common.yaml](https://github.com/alphagov/govuk-puppet/blob/7a1aeee8bd675a741b7293d3a8a25beba93bbbf0/hieradata/common.yaml#L633)

Reply:
Turns out, no it wasn't.  So instead of trying to use `hiera` in the template, I'm now trying to set a var in the manifest and use that in the template.